docs/pages/deploying-vclusters/high-availability.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: High Availability
+sidebar_label: High Availability
+---
+
+import HighAvailabilityK3s from '../fragments/high-availability-k3s.mdx';
+import HighAvailabilityK8s from '../fragments/high-availability-k8s.mdx';
+
+vcluster supports high-availability with k3s as well as the vanilla k8s distribution. k0s is currently not supported for high availability setup of vcluster.
+
+## k3s
+
+<HighAvailabilityK3s />
+
+## Vanilla k8s
+
+<HighAvailabilityK8s />
```

Comment:
See my comment on the other PR about this section

https://github.com/loft-sh/vcluster/pull/1160#discussion_r1296909542